### ISSUE TITLE
Improve grid output accessibility for visually impaired users

### ISSUE THREAD

**Issue by leonardder** [microsoft/vscode-mssql#1560]
- MSSQL Extension Version: 1.8
- VSCode Version: 1.41.1
- OS Version: Windows 10 1909

## Steps to Reproduce:

1. With the NVDA screen reader enabled, start Code, connect to a data base and create a new query.
2. Type a basic select query and press ctrl+shift+e to execute it.
3. Press ctrl+m to allow tab to move focus
4. Tab to the grid output

## Expected result
Keyboard navigation behaves like excel, e.g. left/right arrow move through columns, up/down arrow move through rows

## Actual result
The focus ends up at the section that contains the output grid.

See https://www.w3.org/TR/wai-aria-practices/examples/grid/dataGrids.html for example implementations.

Note that in Azure Data Studio, the accessibility is worse. It's not clear to me how this product is related to azure data studio, it seems to have similar functionality.

**Comment by leonardder** [comment]
@isidorn Is this something you might be able to look into?

**Comment by isidorn** [comment]
@leonardder unfortunetly no. VS Code core can not influence this behavior. vscode-mysql is using a webView I believe so the extension is in control how the focus is moving inside the webView. 
And I think this issue makes sense and should be fixed.

If the extension uses some VS Code widget and that is not accessible that is a different story. However I believe this is not the case here.

**Comment by abdellahtamessouct** [comment]
hello

**Comment by abist** [comment]
You should be able to access the grid in both the extension and Azure Data Studio

**Comment by leonardder** [comment]
Thanks for pointing this out. As for azure data studio, I can confirm that the grid is accessible now.
However, the last release of this extension is from march this year. Is there any reliable way to get the most recent version with this fix installed?
